### ISSUE TITLE
App adds the redundant folders if specified wrong repository path

### ISSUE THREAD

**Issue by TanyaEf** [Jaspersoft/jrsh#170]
#### Steps to reproduce:

1. Type in the console

    ```bash
    $> export repository /pub
    ```
2. Press `Tab` to complete the path and get

    ```bash
    $> export repository /public/
    ```
3. Then move cursor to

    ```bash
    $> export repository /public/
                              ^
    ```
4. Add a few `i` letters more to get
    
    ```bash
    $> export repository /publiiiiic/
    ```
5. Then go to the end of line and press `Tab` several times and you will get

    ```bash
    $> export repository /publiiiic/public/public/
    ```

**Comment by Krasnyanskiy** [comment]
Related to the [pull](https://github.com/Jaspersoft/jrsh/pull/174)

**Comment by Krasnyanskiy** [comment]
Fixed in [v2.0.6](https://github.com/Jaspersoft/jrsh/releases/tag/v2.0.6)
